changelogs/unreleased/1859-skriss

```diff
@@ -0,0 +1 @@
+velero install: if `--use-restic` and `--wait` are specified, wait for restic daemonset to be ready
```

Comment:
Do we want to mention the timeout here?

I think we're up to a total of 2 minutes between the deployment and daemonset waits.

Reply:
sure I can add that.

I didn't bother doing anything more complicated like waiting for both concurrently, as it would require more significant refactoring and it didn't seem super-important to be, but open to others' input on it.

Reply:
Yeah, I agree that separate checks with their own timeouts make sense. I'm not sure there's tons of benefit to combining the checks.